client/mapping-sync/src/sql/mod.rs

```diff
@@ -249,12 +257,23 @@ where
 							let common = tree_route.common_block().hash;
 							tx.send(WorkerCommand::Canonicalize {
 								common,
-								enacted,
-								retracted,
+								enacted: enacted.clone(),
+								retracted: retracted.clone(),
 							}).await.ok();
-						}
 
-						tx.send(WorkerCommand::IndexBestBlock(notification.hash)).await.ok();
+							Some(ReorgInfo {
+								common_ancestor: common,
+								retracted,
+								enacted,
+							})
+						} else {
+							None
+						};
+
+						tx.send(WorkerCommand::IndexBestBlock {
+							block_hash: notification.hash,
+							reorg_info,
+						}).await.ok();
```

Comment:
Best blocks should not emit `reorg` information, at this point the `reorg` information can still change.

Reply:
Addressed in https://github.com/polkadot-evm/frontier/pull/1787/commits/8139c693e699a191ee2a519d1a5e330ca1c3c626